src/map/entities/charentity.cpp

```diff
@@ -655,6 +655,23 @@ CItemEquipment* CCharEntity::getEquip(SLOTTYPE slot)
     return item;
 }
 
+std::vector<CItemEquipment*> CCharEntity::getVisibleEquip()
+{
+    std::vector<CItemEquipment*> visibleItems;
+    CItemEquipment*              item = nullptr;
+
+    for (SLOTTYPE slot : { SLOT_HEAD, SLOT_BODY, SLOT_HANDS, SLOT_LEGS, SLOT_FEET })
```

Comment:
Should this include visible weapons?

Reply:
Unfortunately not a thing I tested for :( What I had read stated only Head Body Hands Legs Feet but should have tested any weapons. Won't be able to test that for a few weeks

Reply:
Personally I'm fine with it as is until proven otherwise